variables.tf

```diff
@@ -17,3 +17,8 @@ variable "image" {
 variable "instance_size" {
   default = "BUILD_GENERAL1_SMALL"
 }
+
+variable "buildspec" {
+  default     = ""
+  description = "The build spec declaration to use for this build project's related builds"
```

Comment:
The question is 
if we define in codebuild resource field buildspec="" would it fetch buildspec.yaml from source code?

If not, then we can not just make description.